[![DOI](https://zenodo.org/badge/833782845.svg)](https://zenodo.org/doi/10.5281/zenodo.13323697)

# Genome assemblies and annotations of _Colpophyllia natans_, _Dendrogyra cylindrus_, and _Siderastrea siderea_

![alt text](https://github.com/mistergroot/cnat_dcyl_ssid_genomes/blob/main/banner.png?raw=true)

Here, we make available the code and genomic resources described in [Locatelli and Baums 2024](https://doi.org/10.1101/2024.08.21.608299 ). These files describe the genomic resources for _Colpophyllia natans_, _Dendrogyra cylindrus_, and _Siderastrea siderea_. All tissue samples were collected in Cura√ßao in August of 2022 at Water Factory Reef (_Colpophyllia natans_, August 6th, 2022) and Sea Aquarium Reef (_Dendrogyra cylindrus_, August 12th, 2022, and _Siderastrea siderea_, August 13th, 2022). Assemblies were generated using PacBio HiFi data generated on a Sequel IIe sequencer and assembled using hifiasm_meta.

In `jupyter_notebooks/`, we provide code for genome assembly, gene prediction, and functional annotation in Funannotate. Further, we provide code for figure generation, orthogroup analyses, and phylogenetically significant gene family expansion analyses (CAFE5).

In `orthogroups/`, we provide OrthoFinder summary output which includes hierarchical orthogroups, as well as unassigned genes for each species.

<ins>__If you intend to use these assemblies, please read the following__</ins>: In `resources/`, we provide highly contiguous, PacBio HiFi-derived genome assemblies (`.fa.gz`), proteins (`.proteins.fa.gz`), CDS transcripts (`.cds-transcripts.fa.gz`), mRNA transcripts (`.mrna-transcripts.fa.gz`), gene predictions (`.gff3.gz`), functional annotations (`.annotations_proteinfer.txt.gz`), and masks for repetitive content (`.repeats.bed.gz`). Functional annotation files output by Funannotate have been updated using [ProteInfer](https://elifesciences.org/articles/80942). Many genes do not get assigned functional annotations (PFAM, GO) straight out of Funannotate, so ProteInfer was used to ensure that all genes have some level of functional annotation, even if it's very high level. All functional annotations inferred with ProteInfer have been indicated as such in the annotation file. To get around GitHub file size restrictions, we provide the genome assemblies and genome annotation files in chunks. A script to prepare assemblies and annotations into single, usable references are included (`prepare_assemblies.sh`). By default, the contigs will not be sorted by size (but their names are ordered by size, e.g. contig_1 is the largest, followed by contig_2, and so forth). If this is something you need, you can do so using `sortbyname.sh` in the `BBmap` suite or another similar option. If you'd like to skip the concatenation steps, the assemblies may be downloaded from NCBI for [_Colpophyllia natans_](https://www.ncbi.nlm.nih.gov/datasets/genome/GCA_043250805.1/), [_Siderastrea siderea_](https://www.ncbi.nlm.nih.gov/datasets/genome/GCA_043250775.1/), and [_Dendrogyra cylindrus_](https://www.ncbi.nlm.nih.gov/datasets/genome/GCA_043250745.1/) and then headers for the annotations can be adjusted using the provided `.GenBank_contig_map.txt` files which link GenBank accessions with contig headers.

The Wellcome Sanger Institute has generated chromosome-level assemblies for [_Siderastrea siderea_](https://www.ncbi.nlm.nih.gov/datasets/genome/GCA_964030785.1/) and [_Dendrogyra cylindrus_](https://www.ncbi.nlm.nih.gov/datasets/genome/GCA_964030785.1/) (_Colpophyllia natans_ is coming but not out yet) sourced from Florida (no publication or gene prediction, yet). We also provide `.chain` and `.agp` files here generated with [RagTag](https://github.com/malonge/RagTag) and [remap-gff3](https://github.com/NAL-i5K/remap-gff3) such that people can update coordinates of any downstream files (e.g. `.vcf` or `.gff3`) presented here or generated elsewhere. This will essentially elevate any analysis performed with our assemblies to chromosome-level. Please ensure that you cite our publication and the [Aquatic Symbiosis Genomics Project](https://wellcomeopenresearch.org/articles/6-254) if you use these files to perform chromosome-level analyses.

Mitochondrial genomes assembled with MitoHiFi are available in both fasta and GenBank (gb) formats with the `.mitogenome.fa` and `.mitogenome.gb` suffixes in the `resources/` directory.

In the assembly notebooks, you may notice that the paths include the name `acer`. The assembly process was very similar to that of _Acropora cervicornis_, described in [Locatelli et al., 2023](https://doi.org/10.1101/2023.12.22.573044). Acer was initially processed with the other species described here, but was spun off into a separate, more in-depth analysis.
